.github/workflows/release_package.yml

```diff
@@ -30,9 +30,22 @@ jobs:
         with:
           node-version: 18
           registry-url: https://npm.pkg.github.com/
-      - run: git config user.email "$GITHUB_ACTOR@gmail.com"
-      - run: git config user.name "$GITHUB_ACTOR"
-      - run: npm version patch
+      - if: $GITHUB_ACTOR == kolbjornkelly
```

Comment:
For nysgjerrighetens skyld:
Ville dette funket? 

- if: $GITHUB_ACTOR == kolbjornkelly || $GITHUB_ACTOR == sarahjelle ... 
  run: git config user.email "$GITHUB_ACTOR@gmail.com"
  run: git config user.name "$GITHUB_ACTOR"
  run: npm version patch

Reply:
Eller failer det fordi mailen blir feil?

Reply:
Eventuelt 
`git config user.name "$(git log -n 1 --pretty=format:%an)"`
`git config user.email "$(git log -n 1 --pretty=format:%ae)"`
... så er man ikke avhengig av gmail